extensions/jackson/deployment/src/main/java/io/quarkus/jackson/deployment/JacksonProcessor.java

```diff
@@ -1,16 +1,65 @@
 package io.quarkus.jackson.deployment;
 
+import java.util.Collection;
+
+import javax.inject.Inject;
+
+import org.jboss.jandex.*;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
 import io.quarkus.deployment.annotations.BuildProducer;
 import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
 import io.quarkus.deployment.builditem.substrate.ReflectiveClassBuildItem;
 
 public class JacksonProcessor {
 
+    private static final DotName JSON_DESERIALIZE = DotName.createSimple(JsonDeserialize.class.getName());
+    private static final DotName BUILDER_VOID = DotName.createSimple(Void.class.getName());
+
+    private static final String JSON_DESERIALIZE_FIELD_BUILDER = "builder";
```

Comment:
I don't think this adds much value. I would prefer it be "inlined" in it's single usage.
The change would make the style similar to what other extensions do

Reply:
Replacing JSON_DESERIALIZE_FIELD_BUILDER with the string "builder" is fine with me. 

However, I prefer both DotNames as they are. For me this is about readability. In my opinion
```
index.getAnnotations(JSON_DESERIALIZE)
```
reads better than
```
index.getAnnotations(DotName.createSimple(JsonDeserialize.class.getName()))
```
Besides that, the BUILDER_VOID is inside a `for`-loop. I prefer not to rely on any compiler optimazation.

Reply:
Yes, I was referring to the "builder" string only